ietf/doc/expire.py

```diff
@@ -225,6 +225,13 @@ def move_file_to(subdir):
             mark = Path(settings.FTP_DIR) / "internet-drafts" / basename
             if mark.exists():
                 mark.unlink()
+            if ext:
+                # Note that we're not moving these strays anywhere - the assumption
+                # is that the active-draft blobstore will not get strays.
+                # See, however, the note about "major system failures" at "unknown_ids"
+                blobname = f"{ext[1:]}/{basename}"
+                if exists_in_storage("active-draft", blobname):
+                    remove_from_storage("active-draft", blobname)
```

Comment:
Just call `remove_from_storage()` and handle an exception if it complains about blobs that don't exist? The code needs to handle that possibility anyway, e.g., due to a race.

Reply:
The implemenation in django-storages (and in boto3) explicitly say it is not an error to remove something that doesn't exist.